packages/Uploader/README.md

```diff
@@ -33,9 +33,10 @@ npm install @paprika/uploader
 | hasAutoUpload      | bool    | false    | true              | On true will upload the file as soon they are selected or dropped                                                                                                                                                                             |
 | isBodyDroppable    | bool    | false    | true              | When true the user will be able to drop files at any part of the document.body. On false will only receive files dropped exactly on the FileInput area.                                                                                       |
 | maxFileSize        | number  | false    | oneMebibyte \* 10 | Size in Mebibytes which is used for comparing each file that will be uploaded.                                                                                                                                                                |
-| onChange           | func    | false    | () => {}          | This callback fires every time a file has been processed                                                                                                                                                                                      |
+| onChange           | func    | false    | () => {}          | This callback fires every time the input value has been changed.                                                                                                                                                                              |
 | onCompleted        | func    | false    | () => {}          | Will fire once all files have been processed with the files as parameter.                                                                                                                                                                     |
 | headers            | arrayOf | false    | []                | you can pass an array of header objects.                                                                                                                                                                                                      |
+| onProcessed        | func    | false    | () => {}          | This callback fires every time a file has been processed.                                                                                                                                                                                     |
```

Comment:
This `onProcessed` is bothering me. It is really just fired once uploading starts… can you update:

```
This callback fires when uploading is about to start (all files have been processed to see if they are valid type/size)
```